deploy/rancher-cleanup.yaml

```diff
@@ -30,9 +30,9 @@ spec:
     spec:
       containers:
       - name: cleanup
-        image: rancher/rancher-cleanup:latest
+        image: ghcr.io/verrazzano/rancher/rancher-cleanup:latest
```

Comment:
latest is usually frowned upon.

Reply:
not sure what this file is for so it might be fine

Reply:
I agree, but that is what the source file has.  I could change it to v1.0.0, maybe that would be better.  For VPO a copy of this file will be in the VPO source and it will get the actual image name and tag from the BOM.